examples/custom_extern/Makefile

```diff
@@ -0,0 +1,32 @@
+P4_PROGRAM = example.json
```

Comment:
that's a bit confusing to call the json file `P4_PROGRAM`
maybe you should use `P4_PROGRAM := example`

Reply:
Done.